android/capacitor/build.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-    androidxAppCompatVersion = project.hasProperty('androidxAppCompatVersion') ? rootProject.ext.androidxAppCompatVersion : '1.1.0'
+    androidxAppCompatVersion = project.hasProperty('androidxAppCompatVersion') ? rootProject.ext.androidxAppCompatVersion : '1.2.0'
```

Comment:
this is a plugin fix, should only contain changes related to the plugin

We also need to update the default version here, but should be a separate PR that updates the version here, and in variables.gradle file, and should be documented that users should update their own variables.gradle file as it's managed by them.
And probably check all other dependencies and update all of them at once, but checking that nothing breaks.

Reply:
So we want a PR that only updates the version in the plugin template here, and then a separate PR that only updates the version in capacitor android and variables.gradle?